### PR TITLE
Prepare for 14.0.0~pre3 Release

### PR DESCRIPTION
**Pull request by azeey**
# 🎈 Release

Preparation for 14.0.0~pre3 release.

Comparison to 13.6.0: https://github.com/gazebosim/sdformat/compare/sdformat13_13.6.0...sdf14

## Checklist
- [ ] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by scpeters** [comment]
do we want to wait for https://github.com/gazebosim/sdformat/pull/1325 and/or https://github.com/gazebosim/sdformat/pull/1330?

**Comment by azeey** [comment]
> do we want to wait for #1325 and/or #1330?

Okay, let's wait for #1325.

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,66 @@
...
+    * [Pull request #1298](https://github.com/gazebosim/sdformat/pull/1298)
+
+1. Make sure the deprecated kSdfScopeDelimiter still works
+    * [Pull request #1305](https://github.com/gazebosim/sdformat/pull/1305)
+
+1. Python script to replace ruby xmlschema generator
+    * [Pull request #1232](https://github.com/gazebosim/sdformat/pull/1232)
+
+1. embedSdf.py: use posix paths
+    * [Pull request #1297](https://github.com/gazebosim/sdformat/pull/1297)
```
Comment (scpeters) [inline Changelog.md:44]: nit: this PR can be added under #884 below

Comment (azeey) [reply Changelog.md]: Done in https://github.com/gazebosim/sdformat/pull/1331/commits/294fbbffb6bf3d7dce6209e82f5f3c1feb06c250

**Review by mjcarroll — APPROVED** [review]

**Review by scpeters — APPROVED** [review]

**Pull request merged**